src/prommis/membrane_examples/ro/tests/test_ro_brine.py

```diff
@@ -0,0 +1,37 @@
+#################################################################################
+# adapted from test_nf.py
+#################################################################################
+
+"""
+Tests the reverse osmosis flowsheet
+"""
+
+from pyomo.environ import value
+
+import pytest
+
+from prommis.membrane_examples.ro.ro_brine import main
+
+pytest.importorskip("watertap", reason="WaterTAP dependency not available")
```

Comment:
Sorry for the confusion; the `pytest.importorskip()` call should be placed before (above) the import statement(s) that require WaterTAP (the same applies to the other test file).

```suggestion
pytest.importorskip("watertap", reason="WaterTAP dependency not available")

from prommis.membrane_examples.ro.ro_brine import main
```